docs/amd.rst

```diff
@@ -41,41 +61,57 @@ Arguments
 +---------------------------------------------------+-----------------------------------------------------------------------------------------------------------------+
-| ``mat_init``                                      | Initial estimate for the mean absorption time (only for oral models, default is 0.1)                            |
+| ``b_init``                                        | Initial estimate for the baseline effect (only for pkpd models)                                                 |
 +---------------------------------------------------+-----------------------------------------------------------------------------------------------------------------+
-| ``b_init``                                        | Initial estimate for the baseline effect (only for pkpd models, default is 0.1)                                 |
+| ``emax_init``                                     | Initial estimate for the Emax (only for pkpd models)                                                            |
 +---------------------------------------------------+-----------------------------------------------------------------------------------------------------------------+
-| ``emax_init``                                     | Initial estimate for the Emax (only for pkpd models, default is 0.1)                                            |
+| ``ec50_init``                                     | Initial estimate for the EC50 (only for pkpd models)                                                            |
 +---------------------------------------------------+-----------------------------------------------------------------------------------------------------------------+
-| ``ec50_init``                                     | Initial estimate for the EC50 (only for pkpd models, default is 0.1)                                            |
+| ``met_init``                                      | Initial estimate for the mean equilibration time (only for pkpd models)                                         |
 +---------------------------------------------------+-----------------------------------------------------------------------------------------------------------------+
-| ``met_init``                                      | Initial estimate for the mean equilibration time (only for pkpd models, default is 0.1)                         |
+| ``dv_types``                                      | Dictionary of DV types for multiple DVs (e.g. dv_types = {'target': 2}). Default is None.                       |
+|                                                   | Allowed keys are: 'drug', 'target', 'complex', 'drug_tot' and 'target_tot'. (For TMDD models only)              |
+|                                                   | For more information see :ref:`here<dv_types>`.                                                                 |
```

Comment:
Can't the same logic be applied to the arguments `b_init`, `emax_init`, `ec50_init`, and `met_init` which are only applied to PKPD models?
I see your point in moving the arguments to their respective page but it feels weird to only remove some of the arguments in my opinion. Perhaps the "model type specific arguments" should only be located on their respective page instead

Reply:
Sorry, I was a bit unclear! I meant that the argument itself can maybe be kept here, but on the TMDD page for example it is written which are allowed. Was thinking if we have more model types it will be pretty dense to have allowed keys for each type on the main page.

Reply:
Aaaah! Ok! Absolutely, I can reduce the description the general page!